pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -141,6 +142,10 @@ var removedPluginsByVersion = []removedPlugins{
 			"RequestedToCapacityRatio",
 		},
 	},
+	{
+		schemeGroupVersion: v1beta3.SchemeGroupVersion.String(),
+		plugins:            []string{},
```

Comment:
Can you make sure in the unit test that the plugins deprecated in v1beta2 are still not supported in v1beta3?

Reply:
The test is already available [here](https://github.com/kubernetes/kubernetes/pull/104251/files#diff-673b4a5dbdb1581c035cc88065bd9a36741b42fce8bb4302fbfeba9ca0a62ea6R366). I switched the version in the validation test which I forgot to push last time.